test/integration/test_uv_init.c

```diff
@@ -130,21 +130,30 @@ TEST(init, dirTooLong, setUp, tearDown, 0, NULL)
     return 0;
 }
 
-#if defined(RWF_NOWAIT)
-static char *oomHeapFaultDelay[] = {"1", "2", NULL};
-#else
-static char *oomHeapFaultDelay[] = {"1", NULL};
```

Comment:
Because when RWF_NOWAIT is **not** available some code paths are not taken, and can't fail with OOM. So in that case setting a delay of `2` (meaning that the second `raft_malloc()` call would fail) would **not** lead to an error, because no `raft_malloc()` would be called just once by the underlying code.